webpack.config.js

```diff
@@ -19,6 +18,8 @@ function readJson (filename) {
   return JSON.parse(fs.readFileSync(filename));
 }
 
+const VERSION = Object.keys(readJson('./src/changelog.json'))[0];
```

Comment:
Should we rather read `package.json` for `VERSION`?

Reply:
Ohh good question. Do we want to force ourselves to write a changelog like this? I'll change it to use package json for this PR.